VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -49,7 +49,7 @@
                 android:id="@+id/download_in_progress_bar"
                 android:orientation="vertical"
                 style="@style/download_in_progress_bar_style"
-
+                android:background="@color/cyan_1"
```

Comment:
I think we're trying to converge on the ``edx_brand_primary_*`` colors to prepare for theming.